pkgs/applications/graphics/drawing/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, file
+, gobject-introspection
+, pkgconfig
+, appstream-glib
+, meson
+, ninja
+, python3
+, gtk3
+, wrapGAppsHook
+, gettext
+, glib
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drawing";
+  version = "0.4.10";
+
+  src = fetchFromGitHub {
+    owner = "maoschanz";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1i8b7hkv9484c0pr2lag6xkf4mq5b2gwcawppgfkf1l8971zxqik";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+    pkgconfig
+    appstream-glib
+    meson
+    ninja
+    python3
+    gtk3
+    gettext
+    glib
+    python3Packages.wrapPython
+  ];
+
+  buildInputs = [
+    file
+    desktop-file-utils
```

Comment:
this is a nativeBuildInput, that's because it's used for desktop file validation
* https://github.com/maoschanz/drawing/blob/100f5e7d491541f41db8e3b89405d97770e2f3aa/data/meson.build#L10

not needed at runtime

Reply:
ok